plugins/lime-plugin-fbw/src/containers/NetworkForm.js

```diff
@@ -11,11 +11,21 @@ import I18n from 'i18n-js';
 
 class NetworkForm extends Component {
 	_changeName (e){
-		this.setState({ communityName: e.target.value || '' });
+		if(!/[^a-zA-Z.]/.test(e.target.value)) {
+			this.setState({ communityName: e.target.value || '' });
+		} else {
+			e.target.value = this.state.communityName || '';
+			this.props.showNotification(I18n.t('Only letters and dots are allowed'))
```

Comment:
I don't know, I preferred to take them off because I just don't know if it works. I leave numbers in the cases of hostnames where it is common to use them.